### ISSUE TITLE
Test error runs in peek view is unreadable when using Light High Contrast theme

### ISSUE THREAD

**Issue by luabud** [microsoft/vscode#143836]
Testing #143425

Using this repo:  https://github.com/luabud/sample_project

if you open the test explorer, run all tests and navigate to the one that fails (test_increment), the error peek view will open. You can see that the error messages on the nodes are unreadable when in Light High Contrast Mode:

![image](https://user-images.githubusercontent.com/45497113/155433759-1774c63e-6e15-4c22-9e74-8ebb296b6ef7.png)

How it looks like in the Light+ theme:
![image](https://user-images.githubusercontent.com/45497113/155433791-89e9b74b-ef14-47bd-b324-7824bb1717c0.png)

**Comment by daviddossett** [comment]
Fixed 🙏 Let me know if you spot any other issues!

https://user-images.githubusercontent.com/25163139/157091345-6e6e123e-4551-4e93-8d93-daba223ee8ac.mp4
